### PR TITLE
Rename to skbeam

### PR DESCRIPTION
**Pull request by ericdill**
@licode This should take care of the rename.  Can you test this?  I do not have any data with which to test pyxrf.  Also note that this has some changes to setup.py and the conda recipe.  See the commit notes on the second commit for the reason why.

### CODE REVIEW

**Comment by licode** [comment]
awesome, I will check this.

File: setup.py [hunk 1]
```diff
@@ -25,7 +22,6 @@
     entry_points={'console_scripts': ['pyxrf = pyxrf.gui:run']},
     package_data={'pyxrf.view': ['*.enaml'], 'configs': ['*.json']},
     include_package_data=True,
-    install_requires=['scikit-xray', 'matplotlib', 'enaml', 'six', 'numpy'],
```
Comment (licode) [inline setup.py]: why remove those?

Comment (ericdill) [inline setup.py]: Did you see the notes I left in the commit message for 6cf1c3d?

Comment (licode) [inline setup.py]: got it. thanks. So if using conda-recipe, we will remove install_requires most of the time.

Comment (ericdill) [inline setup.py]: Well, install_requires should match the build time deps of the conda recipe. I'll fix that now

**Comment by ericdill** [comment]
I misunderstood @licode in our slack channel. He has already done this!

File: conda-recipe/meta.yaml [hunk 2]
```diff
@@ -14,20 +14,17 @@ build:
 requirements:
   build:
     - python
-    - enaml
```
Comment (licode) [inline conda-recipe/meta.yaml]: I thought we need to build enaml?

Comment (ericdill) [inline conda-recipe/meta.yaml]: the things in the `build` section are what `pyxrf` needs to build itself.  This does not build other libraries

**Pull request closed without merging**